java/README.md

```diff
@@ -38,7 +38,7 @@ In some cases there may be a classifier to indicate the version of cuda required
 [Build From Source](#build-from-source) section below for more information about when this
 can happen. No official release of the jar will have a classifier on it.
 
-CUDA 12.0:
+CUDA 12.2:
```

Comment:
Is this all that needs to change for Java? Or are there other things that need to change?

Reply:
Yep had asked about this offline